package.json

```diff
@@ -21,6 +21,7 @@
       "github": "https://github.com/bcasey"
     }
   ],
+  "bin": "./bin/index.js",
```

Comment:
It only runs on `node` which supports all the features it utilizes. I'm not sure how useful it is to support a version of node older than 4.